roles/kubernetes-apps/cluster_roles/tasks/main.yml

```diff
@@ -128,3 +128,28 @@
     - kube_version | version_compare('v1.9.3', '<=')
     - inventory_hostname == groups['kube-master'][0]
   tags: vsphere
+
```

Comment:
Please move this to a new tasks file and use `include_tasks`
Then you can add the `when` condition just one time for each of the tasks under the include_tasks

Reply:
Done